<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DietManagement</class>
 <widget class="QWidget" name="DietManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 lightgreen, stop:1 darkgreen);
}
</string>
  </property>
  <widget class="QLabel" name="Dietmanagement">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; font-style:italic;&quot;&gt;Diet Management Plan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BreakfastPlan">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Breakfast Plan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LunchPlan">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>170</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Lunch Plan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DinnerPlan">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>240</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Dinner Plan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BacktoTabsfromDiet">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Tabs!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
